Client/Clients/UserClient.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using BCrypt.Net;
+using Client.Interfaces;
+using Newtonsoft.Json;
+
+namespace Client.Clients
+{
+    public class UserClient
+    {
+        private string Password { get; set; }
+        private string _userName;
+        private IHttpService _httpService;
+        public string Token { get; private set; }
```

Comment:
Bare for at være sikker, er det ikke meningen at Token skal gemmes uden for UserClient? (I Client State Manager opgaven)

Reply:
Tænker den skal kunne bruges der, men tænker ikke det er statemanagerns opgave at få holde styr på tokenen.